tests/Controllers/ForumTopicsControllerTest.php

```diff
@@ -3,310 +3,192 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
 // See the LICENCE file in the repository root for full licence text.
 
+declare(strict_types=1);
+
 namespace Tests\Controllers;
 
-use App\Models\Forum;
+use App\Models\Forum\Authorize;
+use App\Models\Forum\Forum;
+use App\Models\Forum\Post;
+use App\Models\Forum\Topic;
+use App\Models\Forum\TopicTrack;
 use App\Models\User;
 use App\Models\UserStatistics\Osu as StatisticsOsu;
 use Tests\TestCase;
 
 class ForumTopicsControllerTest extends TestCase
 {
-    public function testDestroy()
```

Comment:
where did this test go?

Reply:
it was a duplicate of testReply with different name, so I removed

...but since this PR is just general cleanup/improvement of forum tests now, I'll write a new one